lib/router/mbaas-spec.js

```diff
@@ -0,0 +1,80 @@
+const assert = require('assert');
+const mbaasRouter = require('./mbaas');
+const express = require('express');
+const supertest = require('supertest');
+const bodyParser = require('body-parser');
+const mediatorMock = require('./mocks/mediatorMock');
+const sampleUser = require('./fixtures/user');
+const sinon = require('sinon');
+
+describe('Test mbass authentication', function() {
+  var app, request;
+
+
+  beforeEach(function() {
+    mediatorMock.request.reset();
+    app = express();
+    app.use(bodyParser.json());
+    mbaasRouter(mediatorMock, app);
+    request = supertest(app);
+
+  });
+
+  it('Should log in using correct credentials', function(done) {
+    request
+      .get('/api/wfm/user/auth')
+      .send({userId: sampleUser.username, password: sampleUser.password})
+      .expect(200, function(err, res) {
+        assert.ok(!err, 'Error on valid credentials ' + err);
+        assert.ok(res, "Expected a result from the authentication request.");
+        assert.equal(res.body.status, 'ok', "Expected status ok from the successful authentication request.");
+        assert.equal(res.body.userId, sampleUser.username, "Expected user profile from the successful authentication request.");
```

Comment:
`sampleUser.username` is used to auth (userIdentifier) and that is expected as `userId` here https://github.com/witmicko/raincatcher-user/blob/0a7d821174219ec58216fb8739ddbd7ba1dd5190/lib/router/mbaas.js#L13
would you prefer to add `userIdentifier` field to the sampleUser fixture?